packages/sign-client/src/controllers/engine.ts

```diff
@@ -1434,6 +1436,7 @@ export class Engine extends IEngine {
         id,
         topic,
         error: err,
+        rpcOpts: ENGINE_RPC_OPTS.wc_sessionPropose.autoReject,
```

Comment:
Is there any potential for breakage with pre-rejection-tags clients when receiving this new tag instead of the default  (https://github.com/WalletConnect/walletconnect-monorepo/pull/4527/files#diff-04e05e19b9270d1267b68d324e16997e4465d6879324fe4427a6194c00410eabL1234) that was used until now?

Reply:
nope, the tags are used internally and have no impact over `client <-> client` behaviour